docs/usage_statistics_exports.md

```diff
@@ -0,0 +1,25 @@
+# How can you export data
+
+Aggregated anonymous daily data can be extracted from the application for any
+specified time period using the `/data-export` route.  Output is available in
+CSV format.
+
+For example, to get data for a five day period starting 10th April 2020:
+
+     <path-to-service>/data-export.csv?start_date=2020-04-08&end_date=2020-04-15
+
+If a date is not supplied it will output a CSV covering all time periods:
```

Comment:
It looks like the default parameters (https://github.com/alphagov/govuk-coronavirus-find-support/pull/185/files#diff-af972ce9990665f118a3e71481574974R19-R20) are from 2020-01-01 to 86400 seconds from now (i.e. 24 hours time)

Reply:
Have amended the docs to clarify, let me know what you think?